alvr/xtask/firewall/alvr_fw_config.sh

```diff
@@ -60,6 +60,10 @@ iptables_cfg() {
     second_port_match_count=$(iptables -S | grep -c '9944')
     if [ "${1}" == 'add' ]; then
         if [ "$first_port_match_count" == "0" ] || [ "$second_port_match_count" == "0" ]; then
+            if [ ! -d '/etc/iptables' ]; then
+                mkdir '/etc/iptables'
```

Comment:
Not too sure about creating it with user permissions. Don't know if it actually matters tho

Reply:
Also I'm not sure we should be creating it if it doesn't exist its probably better to just warn with a message saying iptables are not setup on this distro

Reply:
Tbh if iptables actually being used then I think it's perfectly fine to create the directory, since there shouldn't really be a reason why it doesn't exist, so it can't hurt creating it?

Reply:
We ask for root permission/user at the start of the script, so I don't think it would be created under the user? But I can see the reasoning for just popping out a warning rather than creating the folder implicitly.

Reply:
Yea ur right, I should've actually looked at what the script's doing first, dunno why I assumed it wasn't doing permission elevation even if it clearly wouldn't work if it wasn't doing that. (Also I'm pretty sure you couldn't create the directory anyway if you weren't root.) Then I guess in that regard it's fine, not sure about whether just creating the directory or giving a warning for it is better tho

Reply:
i think this is least intrusive operation, might be fine